.github/workflows/prerelease.yml

```diff
@@ -85,9 +85,8 @@ jobs:
           flavor: |
             latest=false
           tags: |
-            type=semver,pattern={{version}}+py${{ matrix.python-version }}
```

Comment:
`{{version}}-py3.8` for example would always output `1.2.3-alpha.1` because `1.2.3-alpha.1-py3.8` is not valid SemVer, but Docker doesn't support the valid SemVer of `1.2.3-alpha.1+py3.9` (see docs: https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions, https://docs.docker.com/engine/reference/commandline/tag/#description)